### PR TITLE
Switch source directory from java to kotlin

### PR DESCRIPTION
**Pull request by vbuberen**
## :page_facing_up: Context
We don't have Java in the library for quite some time and since nowadays we can more or less safely use `kotlin` folder as a source set I did the change to get rid of `java` folder.

## :pencil: Changes
- Change source sets to use `kotlin` folder instead of `java`
- Fixed lint warning about ViewBinding property

## :no_entry_sign: Breaking
Nothing breaking expected. `apiCheck` task worked fine locally for me after the change.

## :hammer_and_wrench: How to test
Chucker and its tests should work as before without any issues.

### CODE REVIEW

File: library-no-op/build.gradle [hunk 1]
```diff
@@ -11,6 +11,10 @@ android {
         ]
     }
 
+    sourceSets.all {
+        kotlin.srcDir("src/$name/kotlin")
+    }
```
Comment (MiSikora) [inline library-no-op/build.gradle:16]: Is it necessary? If I recall correctly they recently changed AGP to automatically include `kotlin` source sets.

**Review by MiSikora — APPROVED** [review]

File: library-no-op/build.gradle (hunk 1 above)
Comment (vbuberen) [reply library-no-op/build.gradle:16]: I also remember that AGP should handle that, but didn't check. Was keeping in mind that back in spring had to add such construction.

Comment (vbuberen) [reply library-no-op/build.gradle:16]: Will check if it works without and update PR, thanks.

Comment (vbuberen) [reply library-no-op/build.gradle]: Yeah, looks like everything works without adding `sourceSets` lines in `build.gradle` files. Updated.

**Pull request merged**